x-pack/test/api_integration/apis/security/users.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { Cookie, cookie } from 'request';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getService }: FtrProviderContext) {
+  const supertest = getService('supertestWithoutAuth');
+  const security = getService('security');
+  const config = getService('config');
+  const es = getService('es');
+
+  const kibanaServerConfig = config.get('servers.kibana');
+  const validUsername = kibanaServerConfig.username;
+  const validPassword = kibanaServerConfig.password;
+
+  const mockUserName = 'test-user';
+  const mockUserPassword = 'test-password';
+
+  describe('Users', () => {
+    let sessionCookie: Cookie;
+    before(async () => {
+      const loginResponse = await supertest
```

Comment:
I think you can simplify the test setup here. Since we don't need to exercise any session-related code, we can instead use the regular (automatically-authenticated) `supertest` service above, instead of `supertestWithoutAuth`:

```ts
const supertest = getService('supertest')
```

Then we can remove this session cookie code, and in the tests below, we can omit the calls to `.set('Cookie', sessionCookie.toString())`.